web/src/main/java/com/epam/rft/atsy/web/controllers/ApplicationController.java

```diff
@@ -6,12 +6,21 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.servlet.ModelAndView;
 
+/**
+ * This controller handles the non-interactive application page. There is no working logic, the
+ * application page only shows information about
```

Comment:
The last sentence is like it's missing it's end.

It could be: "This page is only used to show information about an application."